### DISCUSSION TITLE
[Feat/Question] Is it possible to add a custom media query (custom prefix) based on an env variable?

### DISCUSSION THREAD

**Discussion by vesper8** [tailwindlabs/tailwindcss#2350] [Ideas]
I'm developing a hybrid app with vue and cordova and I'm having to deal with a lot of instances where there are minor differences in tailwind classes based on whether I'm on IOS or Android. Having to constantly check for isAndroid / isIOS in computed properties is very tedious and verbose.. it would be so awesome if I could do `<div class="ios:tw-pt-18 android:tw-pt-14" />`

I know which platform I'm on based on `process.env.CORDOVA_PLATFORM` which is probably accessible inside tailwind.config.js (unsure). Is there any way to create a custom prefix in tailwind that would allow me to write the `div` sample above? Maybe a plugin already exists?

Thanks~!

**Comment by timedtext** [comment]
Hi, check out [this lib](https://github.com/jaredh159/tailwind-react-native-classnames)

**Reply by vesper8** [reply]
@langtube Thank you that looks awesome! Love the platform and orientation prefixes. But I don't use React.. I wonder if I can still use this.
